### PR TITLE
ClassRoom API - Adding topic

### PR DESCRIPTION
**Pull request by AnnaDinaburgVulikh**
Hi @shiramax , added the option for adding topics and retrieving existing ones.
Also updated the README accordingly.
Should I continue to adding ClassWork functionality?

### CODE REVIEW

File: classroom/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+        `python rose_class.py --topic
+
+For creating a topic in a course use (have to specify the course ID):  
+
+        `python rose_class.py --course -i --topic -c "topic name"`
+
+For retrieving existing topics in a course and their IDs use (have to specify the course ID):  
+
+        `python rose_class.py --course -i --topic -p`
```
Comment (shiramax) [inline classroom/README.md:62]:
```suggestion
        `python rose_class.py --course -i ID --topic -p`
```

File: classroom/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+- Only one invitation can exist for each mail. Before sending a new invitation the old one must be canceled.
+
+### Topics management:  
+Each command will start with:  
+
+        `python rose_class.py --topic
+
+For creating a topic in a course use (have to specify the course ID):  
+
+        `python rose_class.py --course -i --topic -c "topic name"`
```
Comment (shiramax) [inline classroom/README.md:58]:
```suggestion
        `python rose_class.py --course -i ID --topic -c "topic name"`

```

**Review by shiramax — CHANGES_REQUESTED** [review]
@AnnaDinaburgVulikh Looks great as well, minor changes are needed. 
thanks for your contribution.

**Comment by AnnaDinaburgVulikh** [comment]
Hi @shiramax , made the fixes. Thanks for the review.

**Comment by shiramax** [comment]
@AnnaDinaburgVulikh  Can you please rebase?

**Comment by AnnaDinaburgVulikh** [comment]
Hi @shiramax , rebased and also added a check for valid ID in order to have a nice message instead of an error.

**Review by shiramax — APPROVED** [review]
thanks @AnnaDinaburgVulikh great JOB!

**Pull request merged**